pazi.1

```diff
@@ -0,0 +1,122 @@
+.\" Automatically generated by Pandoc 2.9.2
+.\"
+.TH "PAZI" "1" "" "Version 0.4.1" "An autojump \[lq]zap to directory\[rq] helper"
+.hy
+.SH SYNOPSIS
+.PP
+A fast autojump tool
+.PP
+USAGE:
+.IP
+.nf
+\f[C]
+pazi [FLAGS] [OPTIONS] [SUBCOMMAND]
+\f[R]
+.fi
+.PP
+FLAGS:
+.IP
+.nf
+\f[C]
+-h, --help           Prints help information
+
+-i, --interactive    interactively search directory matches
+
+-V, --version        Prints version information
+\f[R]
+.fi
+.PP
+OPTIONS:
+.IP
+.nf
+\f[C]
+    --debug <debug>    print debug information to stderr [env: PAZI_DEBUG=]
+\f[R]
+.fi
+.PP
+SUBCOMMANDS: edit Edit the frecency database
+.IP
+.nf
+\f[C]
+help      Prints this message or the help of the given subcommand(s)
+
+import    Import from another autojump program
+
+init      Prints initialization logic for the given shell to eval
+
+view      View the frecency database
+\f[R]
+.fi
+.SH DESCRIPTION
+.PP
+Pazi is an autojump utility.
+That is to say, pazi remembers visited directories in the past and makes
+it easier to get back to them.
+.PP
+There are several autojump utilities, including fasd (or a better
```

Comment:
The '(or a better maintained fork)' bit doesn't really work without the hrefs.

I think the man page might read more cleanly without this paragraph entirely.